qa/wildfly/build.gradle

```diff
@@ -87,7 +89,7 @@ task deploy(type: Copy) {
     into "${wildflyInstall}/standalone/deployments"
 }
 
-task writeElasticsearchProperties {
+task writeElasticsearchProperties(type: DefaultTestClustersTask) {
```

Comment:
As mentioned, this is actually unnecessary since this is not an incremental task. So currently there's no benefit to tracking these inputs. If we're going to pay the cost of snapshotting these inputs we might as well make the task incremental while we are at it.

Reply:
This is necessary to be able to use the cluster as random tasks can no longer do so. 
We can't make it incremental because  we need to write the port to a file that changes on each run to be able to connect to it lather.